site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -189,6 +245,31 @@ export const CreateWorkspacePageView: FC<CreateWorkspacePageViewProps> = ({
 					</Alert>
 				)}
 
+				{presets.length > 0 && (
+					<FormSection
```

Comment:
A form section should be used to group similar fields. I'm usually don't like to have sections with a single field 🤔 but I will leave this up to you. I was wondering if this could not be in the General section.

Reply:
Understood. There is another issue open to consider the UX and design of this entire page. I have made this change as requested, but I would like to request that we defer further design discussions for the mockups.